### DISCUSSION TITLE
Gutenberg template builder

### DISCUSSION THREAD

**Discussion by sc0ttkclark** [pods-framework/pods#6682] [Ideas]
Be able to build a Gutenberg template (for a post type) using Gutenberg itself.

- [ ] Can add, remove, and move blocks via Gutenberg
- [ ] Unable to edit various settings or edit content of blocks (nice to have, but templates can't take on block initial settings from what I gather so the settings would just go away and not be shown on next template re-edit)

**Comment by ladislavsulc** [comment]
Hello. I could help with my perspective regarding UX/UI but I need more information. Is there any concept at the moment? Any limits? How does your current interaction design workflow works? I could collaborate using Figma.com.

**Comment by sc0ttkclark** [comment]
No concept yet, but the best way for you to get into it would be to play with gutenberg a little bit and try to pretend that instead of laying out content for a *post*, that you're actually laying out a template for a *post type* to use that is then filled in with content by the person using the post editor.

**Comment by benallfree** [comment]
~~~Gutenberg is intended primarily to be a content editor, not a layout editor. There is an extensive discussion around supporting nested blocks in Gutenberg and the consensus seems to be that this is outside the scope of the Gutenberg project: https://github.com/WordPress/gutenberg/issues/219.~~~

edit: nevermind :D

**Comment by lkraav** [comment]
@benallfree it's definitely in (active!) scope, as evidenced by https://github.com/WordPress/gutenberg/pull/3745

Nested blocks is the only way forward, indeed.

**Comment by quasel** [comment]
Well - https://wordpress.org/gutenberg/handbook/templates/#custom-post-types sound like Kind of Layout Templates for me ...

**Comment by ladislavsulc** [comment]
Hi, I was looking into Gutenberg and I have to admit from UX perspective I am not very fond of it. I actually have no clue how to incorporate Pods at this moment other than making Pod item accessible as a block. Also no clue about the layout options, at the moment it seems that each component has to be designed that way and they allow only text columns.

Does anyone have an idea how to approach this?

**Comment by NicoHood** [comment]
Any updates on this one?

**Comment by sc0ttkclark** [comment]
I still want to do this but Pods 2.8 is the current focus. After that is released then our big Gutenberg block editor functionality will be out there and we can start to look at implementations like this.
